src/darwin/Framework/CHIP/MTRCluster.mm

```diff
@@ -111,6 +118,9 @@ - (void)toReadPrepareParams:(chip::app::ReadPrepareParams &)readPrepareParams
     readPrepareParams.mMinIntervalFloorSeconds = self.minInterval.unsignedShortValue;
     readPrepareParams.mMaxIntervalCeilingSeconds = self.maxInterval.unsignedShortValue;
     readPrepareParams.mKeepSubscriptions = !self.replaceExistingSubscriptions;
+    if (self.minimumEventNumber) {
+        readPrepareParams.mEventNumber.SetValue(static_cast<chip::EventNumber>([self.minimumEventNumber unsignedLongLongValue]));
+    }
```

Comment:
This is handled by the super call above, no?  Sorry, I should have caught this last time.

Reply:
@bzbarsky-apple we removed the code because it is duplicated as you mentioned.